lets-talk-bitcoin-podcast/2016-12-25-christopher-jeffrey-consensus-barnacles.md

```diff
@@ -1,18 +1,16 @@
 ---
-title: Christopher Jeffrey Consensus Barnacles (2016-12-25)
+title: Consensus Barnacles
 transcript_by: Michael Folkson
 categories: ['podcast']
 tags: ['consensus']
+date: 2016-12-25
+speakers: ['Christopher Jeffrey']
 ---
 
-Name: Christopher Jeffrey (JJ)
-
-Topic: Barnacles of Consensus
+# Barnacles of Consensus
```

Comment:
Same here.
![Screenshot 2022-11-15 at 4 07 22 PM](https://user-images.githubusercontent.com/18506343/201939756-54217725-cdc0-4e40-83e8-ad7cd4aeae26.png)

It would make more sense to just change the title to this, if the topic gives more context